ci/test.xsh

```diff
@@ -18,7 +18,7 @@ src/bin/lpython --show-cpp tests/doconcurrentloop_01.py
 if $WIN == "1":
     python run_tests.py --skip-run-with-dbg --no-color
 else:
-    python run_tests.py
+    python run_tests.py --skip-run-with-dbg
```

Comment:
Seems like there are currently `5` debug reference tests. These are for now skipped.